spyder/plugins/switcher/widgets/switcher.py

```diff
@@ -293,14 +317,21 @@ def setup(self):
 
         # Filter by text
         titles = []
-        for row in range(self.model.rowCount()):
+        items_data = []
+        for row in range(self.model.rowCount()-1, -1, -1):
```

Comment:
```suggestion
        for row in range(self.model.rowCount() - 1, -1, -1):
```

Adding spaces for pep8.